src/main.cpp

```diff
@@ -2248,16 +2252,38 @@ DisconnectResult DisconnectBlock(const CBlock& block, const CBlockIndex* pindex,
         error("DisconnectBlock(): block and undo data inconsistent");
         return DISCONNECT_FAILED;
     }
+    CAddressIndexAmountVec addressIndex;
+    CAddressUnspentVec addressUnspentIndex;
 
     // undo transactions in reverse order
     for (int i = block.vtx.size() - 1; i >= 0; i--) {
         const CTransaction &tx = block.vtx[i];
-        uint256 hash = tx.GetHash();
+        uint256 const txhash = tx.GetHash();
+
+        if (fUpdateAddressIndex) {
```

Comment:
Instead of introducing a new keyword (or phrase) to search for, I'll make a change so that everything is guarded by `fAddressIndex` (so searching for that will find everything).